### PR TITLE
added ability to sync timer both ways

### PR DESCRIPTION
**Pull request by sohilpandya**
- users can now see the timer running on the watch if started on the phone.
related to #74

### CODE REVIEW

**Review by nelsonic — APPROVED** [review]
@sohilpandya stoked for this feature! 🎉

**Review by iteles — APPROVED** [review]
🎉 LGTM 
Looking forward to testing this tomorrow

File: time/TasksTableViewController.swift [hunk 1]
```diff
@@ -13,37 +13,38 @@ import WatchConnectivity
 
 
 class TasksTableViewController: UITableViewController, WCSessionDelegate {
-    
+
```
Comment (iteles) [inline time/TasksTableViewController.swift:16]: @sohilpandya Are you on your new laptop already? Do you know what caused this spacing diff?

Comment (sohilpandya) [reply time/TasksTableViewController.swift:16]: I am not sure what caused this 😟

**Pull request merged**